### PR TITLE
Created FObjectArrayTableView to display FObject Arrays in a Table View

### PR DESCRIPTION
**Pull request by RichardZhu123**

### CODE REVIEW

**Comment by RichardZhu123** [comment]
Required for https://github.com/nanoPayinc/NANOPAY/pull/7273

File: src/foam/u2/view/FObjectArrayTableView.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+foam.CLASS({
```
Comment (kgrgreer) [inline src/foam/u2/view/FObjectArrayTableView.js:7]: Add copyright header.

File: src/foam/u2/view/FObjectArrayTableView.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    function initE() {
+      var self = this;
+      self.SUPER();
+      self.add(self.slot(function(data) {
+        var dao = self.MDAO.create({
+          of: self.of
+        });
+        data.forEach(d => dao.put(d));
+        console.log(dao.select());
```
Comment (kgrgreer) [inline src/foam/u2/view/FObjectArrayTableView.js]: remove the console.log

File: src/foam/u2/view/FObjectArrayTableView.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      this.of = p.of;
+    },
+    function initE() {
+      var self = this;
+      self.SUPER();
+      self.add(self.slot(function(data) {
+        var dao = self.MDAO.create({
+          of: self.of
+        });
+        data.forEach(d => dao.put(d));
```
Comment (kgrgreer) [inline src/foam/u2/view/FObjectArrayTableView.js:35]: Why do you need to copy the data from 'data' to 'dao'? Can't you just use 'data' directly?

File: src/foam/u2/view/FObjectArrayTableView.js (hunk 1 above)
Comment (RichardZhu123) [reply src/foam/u2/view/FObjectArrayTableView.js:7]: fixed

File: src/foam/u2/view/FObjectArrayTableView.js (hunk 2 above)
Comment (RichardZhu123) [reply src/foam/u2/view/FObjectArrayTableView.js]: fixed

File: src/foam/u2/view/FObjectArrayTableView.js (hunk 3 above)
Comment (RichardZhu123) [reply src/foam/u2/view/FObjectArrayTableView.js:35]: I think the DAOBrowserView expects data to be of DAOProperty, and the data here is just a FObjectArray

**Pull request merged**